staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -2156,6 +2156,31 @@ type VolumeMount struct {
 	// Defaults to false.
 	// +optional
 	ReadOnly bool `json:"readOnly,omitempty" protobuf:"varint,2,opt,name=readOnly"`
+	// RecursiveReadOnly specifies recursive-readonly mode.
+	//
+	// 1. If ReadOnly is false, RecursiveReadOnly must be unspecified.
```

Comment:
For API docs, we should turn this into plain english.  Something like:

```
RecursiveReadOnly specifies whether read-only mounts should be handled
recursively.

If ReadOnly is false, this field has no meaning and must be unspecified.

If ReadOnly is true, and this field is set to Disabled, the mount is not made
recursively read-only.  If this field is set to IfPossible, the mount is made
recursively read-only, if it is supported by the container runtime.  If this
field is set to Enabled, the mount is made recursively read-only if it is
supported by the container runtime, otherwise the pod will not be started and
an error will be generated to indicate the reason.

If this field is set to IfPossible or Enabled, MountPropagation must be set to
None (or be unspecified, which defaults to None).

If this field is not specified on a Pod it defaults to Disabled for backwards
compatibility.  If this field is not specified on an embedded PodSpec (e.g. in
a Deployment), it has no default value until a Pod is created, at which point
it will default to Disabled.
```

Reply:
Applied the suggestion